src/components/sidebar/subMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import { MenuItemProps } from '@/types/sidebarTypes';
+import Link from 'next/link';
+import styles from './subMenu.module.css';
+import { usePathname } from 'next/navigation';
+
+const SubMenu = ({
+  items,
+  selectMenu,
+  selectedItemId,
+}: MenuItemProps & {
+  selectMenu: (id: string) => void;
+  selectedItemId: string | null;
+}) => {
+  const path = usePathname();
```

Comment:
i understand that `const path = usePathname()` is performed on the client side. we also need to consider changing this part to server-side rendering or static when modifying it